### DISCUSSION TITLE
Some experiences and possible improvement UI

### DISCUSSION THREAD

**Discussion by vreijs** [Stellarium/stellarium#2999] [Ideas]
When using Stellarium with sceneries (or are it scenes;-) I have the following experiences:
. When loading a scenery, one sees on the left bottom "Loading scene. Please be patient.". And then on the right bottom 'Loading model...' etc. Would be (semantically) handy to have these two messages at the same location.
. Why use in the above 'scene' and 'model', and not in both cases 'scenery'?
. Would it be possible to Load a scenery (the one that is ticked 'Set as default scene') when starting up?
. In 'Stored views' have a warning before deleting (the 'Delete view' is very close the the other two 'Save view' and 'Add current view'.
. Why use the 'view' (like in 'Stored views') and 'viewpoint' (ie 'Load/save viewpoint' in scenery plugin config) string? Why not 'Viewpoint' in all cases?
. In 'Pointer Coordinates' (plugin): if the cursor is outside the Stellarium window, the values also changes. I would restrict this to only within Stellarium window.
. In 'Point Coordinates' the minute-degree value can become '60'. Some rounding issue?

Hope this helps and if questions, let me know.

All the best in 2023,

Victor

**Comment by gzotti** [comment]
> When using Stellarium with sceneries (or are it scenes;-) I have the following experiences: 
>. When loading a scenery, one sees on the left bottom "Loading scene. Please be patient.". And then on the right bottom 'Loading model...' etc. Would be (semantically) handy to have these two messages at the same location. 

The message on the left was the only hint about loading until IIRC somebody suggested we add the loading bar ... 
The latter has its fixed location. But I think the message is better visible on the left side. 

>. Why use in the above 'scene' and 'model', and not in both cases 'scenery'? 

Scenery = model+landscape. Yes, we could harmonize the messages.

>. Would it be possible to Load a scenery (the one that is ticked 'Set as default scene') when starting up? 

No, the model is loaded on first switching on. Most users will use Stellarium regularly and only occasionally switch on the plugin. Pre-loading a large scene will be experienced as a waste of time. 

>. In 'Stored views' have a warning before deleting (the 'Delete view' is very close the the other two 'Save view' and 'Add current view'. 

Maybe. The button labels speak for themselves, so we assume our users know what is about to happen when pressing "delete". Do you think we really have to confirm each such rather insignificant action with yet another "Are you sure" dialog & YES/NO mouse click? 

>. Why use the 'view' (like in 'Stored views') and 'viewpoint' (ie 'Load/save viewpoint' in scenery plugin config) string? Why not 'Viewpoint' in all cases? 

May be harmonized. 

>. In 'Pointer Coordinates' (plugin): if the cursor is outside the Stellarium window, the values also changes. I would restrict this to only within Stellarium window. 

Maybe possible. 

>. In 'Point Coordinates' the minute-degree value can become '60'. Some rounding issue?

Presumably yes. 
 
> Hope this helps and if questions, let me know.
> 
> All the best in 2023,
> 
> Victor

Thanks for comments. Most little things just went by with nobody complaining...
